content/2017-09-12_python-komunita-a-ja.md

```diff
@@ -0,0 +1,63 @@
+
+Přemýšlel jsem tedy o tom, že by se měla moje role změnit a jasněji určit. Abych věděl, na čem přesně mám pracovat, a co mám pustit z hlavy. Aby i ostatní měli jasno v tom, co ode mě mají očekávat, a co už ne. Takže tady to je. Snažil jsem se před programováním apod. činnostmi, kde jsem nahraditelný, upřednostnit věci, kde mohu uplatnit svůj přehled v komunitě a znalost její historie.
+
+## Čím se chci zabývat
+
+Seřazeno podle priorit.
+
+1. Pomocník Martina Bílka (předseda Pyvce) v tom, aby se Pyvec transformoval z občanského sdružení na spolek a hlavně na něco, co bude lépe škálovat a lépe sloužit současným potřebám komunity.
+2. Majitel domén, který převede [pyvo.cz](https://github.com/pyvec/zapojse/issues/3) a [pyladies.cz](https://github.com/pyvec/zapojse/issues/10) na Pyvec.
+3. Správce a hybatel meta návodů, jako je [Pyvec Guide](https://pyvec-guide.readthedocs.io/), nebo tzv. [welcome guide](https://github.com/pyvec/zapojse/issues/59).
+4. Autor nové verze a správce [pyvec.org].
+5. Člen organizačního týmu PyCon CZ, který má na starost propagaci konference (přičemž tuto roli bych rád pojímal každým rokem více a více mentorsky a předával zkušenosti na ideálně co největší okruh lidí).
+6. Zadavatel (a když se nikdo nenajde, tak i zhotovitel) nástroje pro PyCon CZ, který umožní tweetovat série podobných sdělení v konkrétních časech.
+7. Pisálek, který možná napíše příručku pro propagaci akcí na sociálních sítích.
+8. Správce [python.cz].
+9. Pisálek, který po letech dokončí [příručku pro organizaci srazů](https://github.com/pyvec/zapojse/issues/5).
+10. Pisálek, který má čas napsat i technické články o tom, jak co v Pythonu udělat, ať už na blog nebo někam na [Zdroják.cz](https://www.zdrojak.cz/). Pisálek, který má čas sepsat třeba materiály pro workshop o API apod.
+
+Správce znamená, že bych rád nějak vedl směřování, kterým se webová stánka ubírá a rád bych nad ní měl ochrannou ruku, ale na jejím vývoji by se ideálně podílel tým lidí. Mou prací by také mělo být hledání začátečníků, kteří mohou pomoci se snadnějšími úkoly, a s jejich mentorováním.
+
+## Co bych dělal rád, ale nemám to jak stihnout
+
+K následujícím věcem mám nějaký vztah a bavily by mě, ale řekl jsem si, že mají menší prioritu nebo že jsem v nich zastupitelný. Popřípadě že v nich mohu vystupovat nanejvýš jako poradce, mentor, nebo koordinátor.
+
+- [Překladatel přednášky o mentorování](https://github.com/pyvec/zapojse/issues/63).
+- Hnací síla za zhotovením [job boardu pro PyLadies](https://github.com/pyvec/zapojse/issues/35).
+- Vývojář [blog.python.cz], někdo kdo dokumentuje jeho používání, a opravuje na něm chyby (tzn. issues [pyvec/blog.python.cz](https://github.com/pyvec/blog.python.cz/issues) a [honzajavorek/danube-delta](https://github.com/honzajavorek/danube-delta/issues)).
+
+## Co už nechci dělat
+
+- Organizovat brněnské (a nejspíš ani žádné jiné) Pyvo. Šest let stačilo!
+- Být zodpovědný za propagaci či organizaci jiných věcí, než je PyCon CZ.
```

Comment:
- zrušit čárku před "než"

Reply:
nesouhlasím. než je PyCON CZ -> VV s přísudkem jmenným se sponou.

Reply:
Se neshodnete, tak nechám co je 😄